.travis.yml

```diff
@@ -5,7 +5,16 @@ services:
 
 install: true
 
+env:
+  - DOCKER_COMPOSE_VERSION=1.24.0
+
 before_script:
```

Comment:
Please pick cherries from: https://github.com/sevenval/dockerfiles/blob/b041b69b5c1694f062f179e8e5f65dec9903b88f/.travis.yml

Reply:
I had to copy the stuff.